README.md

```diff
@@ -30,6 +30,27 @@ Current contributors include:
 
 [![Figure 1. _iSEE_ uses a customisable multi-panel layout.][Figure1]](https://f1000research.com/articles/7-741/v1)
 
+## Installation
+
+_iSEE_ can be easily installed from Bioconductor using `BiocManager::install()`:
+
+``` r
```

Comment:
Shouldn't be a space here.

Reply:
I saw it in other repos, works in both cases ;)